src/components/Hello.vue

```diff
@@ -1,7 +1,28 @@
 <template>
-  <div class="hello">
-    <h1>{{ msg }}</h1>
+
+  <div class="park">
+
+
+  
+    <p> I'm interested in joining the mPower study because I </p>
+    <br>
+    <br>
+    <select placeholder= '' id="interest">
+        <option value="empty" disabled selected>Select Answer </option>
+        <option  value="hasParkinsons">have parkinson's disease</option>
+    </select>
+    <br><br>
+   <p v-if="seen"> and I would be willing to try </p>
```

Comment:
These v-if's could be made DRY'er by placing these elements in a `<template v-if="ok"></template>` block, `<div>`, etc.